src/org/omegat/Bundle.properties

```diff
@@ -1507,10 +1507,10 @@ SDLPROJECT_FILTER_NAME=SDL project
 SDLXLIFF_FILTER_NAME=SDL XLIFF
 
 # org.omegat.filters4.xml.xliff.Xliff1Filter.java
-XLIFF1FILTER_FILTER_NAME=XLIFF 1.x Bilingual
+XLIFF1FILTER_FILTER_NAME=XLIFF
 
```

Comment:
I thought you wanted to name them:
- XLIFF 1.x
- XLIFF 2.x

It's probably fine with XLIFF 2 right below, but just XLIFF might be misinterpreted as applying to all versions of the format.

Reply:
OK. In that case this is probably ready to be merged if you're happy with the wording tweaks I made in `App_filter.xml`.